extensions/ql-vscode/src/data-serialization/generated-schemas/RemoteQueryHistoryItem.json

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
We could use `.gitattributes` to have files in this directory be marked as generated. GitHub might work it out by itself but we can still give it a hint to make sure it gets it correct.
https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github

Reply:
Don't we want to see when we add a new field to the schemas?

Reply:
I guess it would be enough to see the changes to associated types... I'll add it.

Reply:
Yeah you should be able to judge the change being made and how compatible it is by looking at the typescript types. They're still in the `data-serialization` directory too.